lib/pom.rb

```diff
@@ -73,8 +73,8 @@ def initialize( name, version, default_spec = true )
   #            :id => 'gem-staging' )
 
   plugin( :clean,
-          :filesets => [ { :directory => '${basedir}/ruby/gems/shared/specifications/default',
-                           :includes => [ '*' ] },
+          :filesets => [ { :directory => '${basedir}/ruby/gems',
```

Comment:
if we move this extra clean into 'dist' profile, then a `mvn clean install -Pdist` is doing the right thing.

there is no need to add new gems in this line https://github.com/jruby/jruby/pull/4083/files#diff-a5692734fb1e233c63e88629e6f7bd12L290 - the bin files can be probably omitted here.